Iceberg-TipUI/IceTipDialog.class.st

```diff
@@ -20,11 +20,19 @@ IceTipDialog >> cancel [
 	
 ]
 
-{ #category : #accessing }
-IceTipDialog >> openDialogWithSpec [
+{ #category : #initialization }
+IceTipDialog >> initializeDialogWindow: aWindow [
+
+	super initializeDialogWindow: aWindow.
 
-	^ super openDialogWithSpec
+	aWindow 
 		  okAction: [ self accept ];
-		  cancelAction: [ self cancel ];
-		  yourself
+		  cancelAction: [ self cancel ].
+]
+
+{ #category : #showing }
+IceTipDialog >> openDialogWithSpec [
+
+	^ self openModalWithSpec
```

Comment:
@tesonep Tests started to fail because of this change. I will create issue to explain it there.

Reply:
#1438